### PR TITLE
Bump the default version 1.13.2

### PR DESCRIPTION
**Pull request by yujuhong**
Using newer kubelet/kube-proxy binaries for testing

### CODE REVIEW

File: cluster/gce/win1803/node-helper.sh [hunk 1]
```diff
@@ -57,7 +57,7 @@ function get-windows-node-instance-metadata {
   # TODO(pjh): check how KUBE_VERSION is set - does it make sense to use for
   # downloading the kubernetes binaries on the node?
   #metadata+="k8s-version=v1.12.0"
-  metadata+="k8s-version=${KUBE_VERSION:-v1.11.3},"
```
Comment (yujuhong) [inline cluster/gce/win1803/node-helper.sh]: I think we should get rid of this. @pjh

Comment (mtaufen) [reply cluster/gce/win1803/node-helper.sh]: SGTM

Comment (pjh) [reply cluster/gce/win1803/node-helper.sh]: Yep.

**Comment by pjh** [comment]
LGTM

**Comment by yujuhong** [comment]
The test job has been fixed. Merging this so that it'd be tested in the next round.

**Pull request merged**